packages/runtime/runtime-definitions/src/dataStoreContext.ts

```diff
@@ -486,12 +496,23 @@ export interface IFluidDataStoreContext
 	 * @deprecated There is no replacement for this, its functionality is no longer needed at this layer.
 	 * It will be removed in a future release, sometime after 2.0.0-internal.8.0.0
 	 *
+	 * Similar capability is exposed with from/to string paths instead of handles via @see addedGCOutboundRoute
+	 *
 	 * Called when a new outbound reference is added to another node. This is used by garbage collection to identify
 	 * all references added in the system.
 	 * @param srcHandle - The handle of the node that added the reference.
 	 * @param outboundHandle - The handle of the outbound node that is referenced.
 	 */
 	addedGCOutboundReference?(srcHandle: IFluidHandle, outboundHandle: IFluidHandle): void;
```

Comment:
I wonder if this can be overloaded

Reply:
Yeah, I thought about that a bit. Expanding the "in" types means someone _using_ this interface sees that as a backwards compatible change (existing calls are fine) but someone _implementing_ the interface sees it as a non-back-compat change. Of course no one implements this, and our type tests don't even cover function signature changes, but I'd rather keep it simple.